daemon/archive.go

```diff
@@ -152,6 +152,22 @@ func (daemon *Daemon) containerStatPath(container *container.Container, path str
 	container.Lock()
 	defer container.Unlock()
 
+	// We must pause the container (if it is running) before doing filesystem
+	// operations in order to prevent active filesystem-based attacks such as
+	// CVE-2018-15564.
```

Comment:
Are you sure referring correct CVE?

https://nvd.nist.gov/vuln/detail/CVE-2018-15564

Reply:
It should be CVE-2018-15664. I fixed the commit messages but forgot to fix the comments.